runtime/src/bank.rs

```diff
@@ -498,6 +547,98 @@ impl Bank {
         new
     }
 
+    /// Create a bank from explicit arguments and deserialized fields from snapshot
+    pub(crate) fn new_from_fields(
+        bank_rc: BankRc,
+        genesis_config: &GenesisConfig,
+        fields: BankFields,
+    ) -> Self {
+        fn new<T: Default>() -> T {
```

Comment:
Heh, I guess this acceptable evacuation. But `new` is so similar to the tranditional `::new()`. The `::new()` can take some arguments to initialize a new instance. At first glance, I'm confused with them.

How about renaming this to something like `default_generic()` or just `default()`?

Reply:
I wanted something short that sounded like a ctor.
"new" is three letters whereas "default"is more than twice that.
But fine to go with the latter if it's clearer.

Reply:
Ok, fine as-is! Come to think of it, `HashMap::new()` is completely legal in rust as well. I've updated my mental model. :)